E2E/assertions/assertions.ts

```diff
@@ -0,0 +1,43 @@
+const { I, game } = inject();
+const complexitySP = [1,2,3,5,8,13];
```

Comment:
using arrays for these kind of values is extremely confusing as `complexitySP[1]` would result to `2`.